docs/PAYLOAD.md

```diff
@@ -16,7 +16,7 @@
   - [Priority in Notifications](#priority-in-notifications)
   - [Picture Messages](#picture-messages)
   - [Background Notifications](#background-notifications)
-    - [Use of content_available: true](#use-of-content-available-true)
+    - Use of content_available: true](#use-of-content_available-true
```

Comment:
@ptandler I believe you accidentally removed the open and closing brackets from this line which will cause the href to be broken